### ISSUE TITLE
Tagging queries without transactions?

### ISSUE THREAD

**Issue by OrKoN** [vitaly-t/pg-monitor#12]
Hi @vitaly-t 

I was wondering if there is a way to tag a query that is not a task or transaction? I would like to provider a context id so that I know what http request has caused the query execution when I log it with pg-monitor. For example:

```
db.none('UPDATE ...', 'my-http-request-id')

... 


const log = (msg, info) => {
  logger.debug(`${info.event}: ${info.text}`, {
    sql: true,
    event: info.event,
    requestId: info.tag,
  });
  info.display = false;
};
```

Thanks in advance

**Comment by vitaly-t** [comment]
Only tasks support tagging. You can always use a task even for a single query, it is no overhead in performance.

Alternatively, you can add your own method via event `extend`, one that would take in a tag and then output it as you like.

**Comment by OrKoN** [comment]
@vitaly-t I have tried the log tagged tasks but it seems that tag is not available on the info object as I have expected. Is there another way to intercept with the logging so that I get the tag a standalone attribute?

**Comment by vitaly-t** [comment]
Event `log` doesn't contain a separate `tag` attribute, because it is part of the formatted message - `msg` property. So if you simply log the `msg`, the tag will be there.

**Comment by vitaly-t** [comment]
I've added an initial commit to expose the entire [ctx = task/transaction context](http://vitaly-t.github.io/pg-promise/Task.html#.ctx) within the `log` event: https://github.com/vitaly-t/pg-monitor/commit/5dc56d8f01a9dbf47544e03a885e77c05ddcc46f

Will finish it later, when I get a chance ;)

**Comment by OrKoN** [comment]
@vitaly-t thanks

**Comment by vitaly-t** [comment]
@OrKoN Finished and released as [v0.8.0](https://github.com/vitaly-t/pg-monitor/releases/tag/v.0.8.0).

Now you can do:

```js
const log = (msg, info) => {
    if (info.ctx) {
        console.log('TAG:', info.ctx.tag);
    }
};
```
